packages/orchestrator/lib/client.integration.test.ts

```diff
@@ -31,94 +31,102 @@ describe('OrchestratorClient', async () => {
         await clearDb();
     });
 
-    it('should schedule immediate task', async () => {
-        const groupKey = 'groupA';
-        const res = (
-            await client.schedule({
-                name: 'Task',
-                groupKey: groupKey,
-                retry: { count: 3, max: 5 },
-                timeoutSettingsInSecs: {
-                    createdToStarted: 10,
-                    startedToCompleted: 10,
-                    heartbeat: 10
-                },
-                args: {
-                    name: 'Action',
-                    connection: {
-                        id: 1234,
-                        provider_config_key: 'P',
-                        environment_id: 5678
+    describe('schedule', () => {
```

Comment:
You can scroll to the end of this file where the only changes is. The rest is just grouping tests